app/views/users/_organizations_area.html.erb

```diff
@@ -1,9 +1,11 @@
+<%# it's highly unlikely a single use will belong to so many organizations to slow down
+    this fetch, thus we can safely use the .present?/.each pattern to load them %>
 <% if @user.organizations.present? %>
   <div id="sidebar-organizations" class="widget">
     <div class="widget-suggested-follows-container">
       <header><h4>organizations</h4></header>
       <div class="widget-body">
-        <% @user.organizations.find_each do |organization| %>
+        <% @user.organizations.each do |organization| %>
```

Comment:
Going this way I am fine with bc I dont think a user will likely every belong to thousands of orgs.